moz_kinto_publisher/main.py

```diff
@@ -145,6 +145,33 @@ def attach_file(
                 + f"{response.content.decode('utf-8')}"
             )
 
+    def request_review_of_collection(self, *, collection=None):
+        try:
+            resp = self.get_collection(id=collection)
+        except KintoException as e:
+            log.error("Couldn't determine {collection} review status")
+            raise e
+
+        original = resp.get("data")
+        if original is None:
+            raise KintoException("Malformed response from Kinto")
+
+        status = original.get("status")
+        if status is None:
+            raise KintoException("Malformed response from Kinto")
+
+        if status != "work-in-progress":
+            log.info("Collection is not marked for review. Skipping.")
```

Comment:
Should this be `... is not marked as in progress` or something?

Reply:
yep. thanks!